paddle/fluid/operators/detail/request_handler.h

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
I've written an example at: https://gist.github.com/typhoonzero/fc2489231c8e3a29d9ac7c895c8f4aae, which shows how to abstract the rpcserver. This implement can actually remove the redundant "blocking queue" also.